tests/cephfs/snapshot_clone/snap_schedule.py

```diff
@@ -109,6 +111,19 @@ def run(ceph_cluster, **kw):
         m_granularity = (
             "m" if LooseVersion(ceph_version) >= LooseVersion("17.2.6") else "M"
         )
+        log.info("Verify Ceph Status is healthy before starting test")
```

Comment:
Can this be converted to a function and added to ceph utils, so that we can make use of this in other tests when required?